README.md

```diff
@@ -569,7 +569,6 @@ function renderSomething(){
 <details>
     <summary>Prettier makes some spacing and the use of quotes and a few other things consistent across the project.</summary>
 
-
```

Comment:
can you remove the change to README.md - I don't think it's intended.  just do `git checkout README.md`